components/installing_remote_computer_software.mdx

```diff
@@ -0,0 +1,69 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+It is often convenient to use a Remote Computer (eg. laptop) to command and observe your robot. To do this, your
```

Comment:
We have used several terms throughout the site:

- Remote PC
- Remote Computer
- Desktop
- Development

It would be nice to have a consistent term to help people search through our webpages.
This could be considered in a separate Pull Request.

Reply:
I'm suggesting Remote Computer, but open to suggestions. It should now be modified to that throughout.